applications/launcher/controller.cpp

```diff
@@ -370,6 +400,44 @@ void Controller::setColumns(int columns){
         emit columnsChanged(columns);
     }
 }
+void Controller::setSwipeLength(int direction, int length){
+    switch (direction){
+        case 1:
+            m_swipeLengthRight = length;
+            emit swipeLengthRightChanged(length);
+            break;
+        case 2:
+            m_swipeLengthLeft = length;
+            emit swipeLengthLeftChanged(length);
+            break;
+        case 3:
+            m_swipeLengthUp = length;
+            emit swipeLengthUpChanged(length);
+            break;
+        case 4:
+            m_swipeLengthDown = length;
+            emit swipeLengthDownChanged(length);
+            break;
+        default:
+            qDebug() << "Invalid swipe direction: " << direction;
+            break;
+    }
+}
+int Controller::getSwipeLength(int direction){
+    switch (direction){
+        case 1:
+            return swipeLengthRight();
+        case 2:
+            return swipeLengthLeft();
+        case 3:
+            return swipeLengthUp();
+        case 4:
+            return swipeLengthDown();
+        default:
+            qDebug() << "Invalid swipe direction: " << direction;
+            return -1;
+    }
+}
```

Comment:
I'd prefer to use an enum here for readability instead of the raw integer values. Similar to what I do for WifiStatus etc: https://github.com/Eeems/oxide/blob/master/applications/launcher/controller.h#L28-L31